### DISCUSSION TITLE
EDC version while running.

### DISCUSSION THREAD

**Discussion by JorgeBarto** [eclipse-edc/Connector#4471] [Q&A] [answered]
Hi everyone,
I would like to know if there is any way to confirm which EDC version is running.
I mean, when I have running a connector (consumer or provider), does it exist an endPoint, a log trace or any other information where I could be able to confirm the EDC version.
I hope I made myself clear enough.
Thanks in advance!!

**Comment by paullatzelsperger** [comment, marked as answer]
nope, such an endpoint does not exist because the EDC version is not relevant at runtime. You can get version information for the DSP API and for all of EDC's REST APIs though.

If you talk to another EDC via DSP, you don't care what that connector's runtime version is, because all you care about is that it behaves according to the DataSpace Protocol (DSP). 

If you want to know the version of _your_ EDC (i.e. that runs in your K8S cluster) then that is a deployment problem, not an EDC problem.

**Reply by JorgeBarto** [reply]
If I run the command: gradle build --INFO I'm able to see traces such as:
**_Transforming edc-build-0.9.0.jar (org.eclipse.edc:edc-build:0.9.0)_**
Does it mean the EDC version?, in this case 0.9.0.
Thanks!

**Reply by paullatzelsperger** [reply]
I'm sorry, I don't understand your question. If you have a Java project, presumably a Gradle project that uses EDC modules, you have to declare your EDC dependencies somewhere, either in your build file or in a version catalog. So unless you are using version wildcards (in which case your build is bad, and you should feel bad :)), you already know what version you are using, because, well, you put it there...?
